### PR TITLE
sync dirty

### PR DESCRIPTION
**Pull request by mattfred**
@frett This should take care of the training sync

### CODE REVIEW

File: app/src/main/java/com/expidev/gcmapp/service/TrainingService.java [hunk 1]
```diff
@@ -91,6 +93,10 @@ protected void onHandleIntent(Intent intent)
                 case SYNC_TRAINING:
                     syncTraining(intent);
                     break;
+                case SYNC_DIRTY_TRAINING:
+                    syncDirtyTraining();
```
Comment (frett) [inline app/src/main/java/com/expidev/gcmapp/service/TrainingService.java]: `break;`

File: app/src/main/java/com/expidev/gcmapp/model/Training.java [hunk 2]
```diff
@@ -32,9 +37,12 @@
     @NonNull
     private final Set<String> mDirty = new HashSet<>();
 
+    public static final String JSON_ID = "id";
     public static final String JSON_NAME = "name";
+    public static final String JSON_MINISTRY_ID = "ministryId";
```
Comment (frett) [inline app/src/main/java/com/expidev/gcmapp/model/Training.java]: should be `ministry_id`

**Comment by frett** [comment]
other than the couple minor fixes it looks good

**Comment by mattfred** [comment]
Ok these are fixed. I will merge.

**Pull request merged**